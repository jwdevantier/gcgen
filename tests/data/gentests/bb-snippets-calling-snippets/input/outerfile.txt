# [[start bar
# end]]

# [[start baz
# end]]
