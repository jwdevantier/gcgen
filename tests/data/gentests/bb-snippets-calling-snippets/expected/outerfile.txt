# <<? bar
outer bar!
outer foo!
# ?>>

# <<? baz
outer baz!
outer foo!
# ?>>
